src/pseudopeople/dataset.py

```diff
@@ -0,0 +1,154 @@
+from pathlib import Path
```

Comment:
Many of the methods in this class were copied over from interface.py or noise.py and then updated to be instance methods.